src/change-hub.js

```diff
@@ -45,7 +59,14 @@ export async function changeHub(hubId, addToHistory = true, waypoint = null) {
   const hub = data.hubs[0];
 
   if (addToHistory) {
-    window.history.pushState(null, null, hubUrl(hubId, {}, hub.slug, waypoint));
+    const search = new URLSearchParams(location.search);
+    const params = {};
+    for (const [key, value] of search.entries()) {
```

Comment:
I assume this was just an unrelated fix?

Reply:
Yes